indexer/packages/postgres/src/db/migrations/migration_files/20240829161445_add_fills_affiliateearnedrevshare.ts

```diff
@@ -2,12 +2,12 @@ import * as Knex from 'knex';
 
 export async function up(knex: Knex): Promise<void> {
   await knex.schema.table('fills', (table) => {
-    table.string('affiliateEarnedRevShare').notNullable().defaultTo('0');
+    table.string('affiliateRevShare').notNullable().defaultTo('0');
```

Comment:
Could you confirm that no deployments have been made since this migration was merged? would be safer to create a new migration otherwise

Reply:
Good point. To be safe I will just create a new migration